sites/inc-skiplinks.html

```diff
@@ -1,6 +1,6 @@
 <nav><ul id="wb-tphp">
 	<li class="wb-slc"><a class="wb-sl" href="#wb-cont">{{ i18nText-skipCont }}</a></li>
-{%- unless page.nofooter -%}
+{%- unless page.nofooterAbout and page.nofooterSite -%}
```

Comment:
Camel case adjustment here as well
```suggestion
{%- unless page.noFooterAbout and page.noFooterSite -%}
```